azkaban-common/src/main/java/azkaban/utils/StdOutErrRedirect.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.utils;
+
+import java.io.PrintStream;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.Level;
+
+public class StdOutErrRedirect {
+  private static final Logger logger = Logger.getLogger(StdOutErrRedirect.class);
+
+  public static void redirectOutAndErrToLog() {
+    System.setOut(createStream(System.out, Level.INFO));
+    System.setErr(createStream(System.err, Level.ERROR));
+  }
+
+  private static PrintStream createStream(PrintStream stream, Level level) {
+    return new PrintStream(stream) {
+      public void println(final String string) {
```

Comment:
Curious: Will this work if I print an integer ?

Best practices: if you are overriding/implementing a function, use annotations to indicate them. Like in this case using @Override.